webapp/public/js/domjudge.js

```diff
@@ -599,6 +602,21 @@ function updateMenuInternalErrors(data)
     }
 }
 
+function updateMenuBalloons(data)
+{
+    var num = data.length;
+    if ( num != 0 ) {
+        for (var i=0; i<num; i++) {
+            var text = (data[i].room !== null) ? data[i].room+': ' : '';
+            text += data[i].pname + ' ' + data[i].name;
+			sendNotification('New balloon:',
+                 {'tag': 'ball_' + data[i].baloonid,
+                  'link': domjudge_base_url + '/jury/balloons',
+                  'body':  text});
```

Comment:
Nitpick: there is a space too much after the :

Reply:
I'll close this merge request, and fix it in another branch